src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitUsingTests.cs

```diff
@@ -39,11 +29,11 @@ public System.Threading.Tasks.ValueTask DisposeAsync()
     }
 }
 ";
-            var comp = CreateCompilationWithTasksExtensions(source + s_interfaces, parseOptions: TestOptions.Regular7_3);
+            var comp = CreateCompilationWithTasksExtensions(source + CSharpTestBase.IAsyncDisposableDefinition, parseOptions: TestOptions.Regular7_3);
```

Comment:
>source + CSharpTestBase.IAsyncDisposableDefinition [](start = 60, length = 50)

Since you're changing all these, consider using `new[] { source, IAsyncDisposableDefinition }` instead if there's no reason to concatenate the two source files.